src/zenml/materializers/pandas_materializer.py

```diff
@@ -23,17 +23,21 @@
 from zenml.logger import get_logger
 from zenml.materializers.base_materializer import BaseMaterializer
 from zenml.metadata.metadata_types import DType, MetadataType
+from zenml.utils import io_utils
 
 logger = get_logger(__name__)
 
-PARQUET_FILENAME = "df.parquet.gzip"
-COMPRESSION_TYPE = "gzip"
+PARQUET_FILENAME = "df.parquet"
```

Comment:
Doesn't this break the functionality where there is an artifact saved as a parquet file with the old materializer, and people upgrade their ZenML version?

In more detail, in this case, the `load` function would check:
- if the metadata file exists and would skip,
- then it would check whether the parquet file exists and since the filename has changed would skip again,
- finally, would try to load a csv file and fail.

Reply:
> then it would check whether the parquet file exists and since the filename has changed would skip again,

It would check if the parquet files exists and would not skip as it would exist

Reply:
Since it was generated by the old materializer, the filepath of the old artifact would be `self.uri/df.parquet.gzip`. But the new materializer will look for it on `self.uri/df.parquet`. or am I missing something?